app/assets/stylesheets/application.scss

```diff
@@ -133,3 +133,9 @@ $govuk-breakpoints: (
     margin: 0 auto;
   }
 }
+
+.raise-a-repair-button {
```

Comment:
Should probably use a standard button rather than a special one. I assume you're following the design, so perhaps it would be a good idea to ask lawrence how important the custom button is

Reply:
I'ts not really a button, even though the name suggest it. This class is mainly to align the text vertically...

Reply:
I just removed that ugly class

Reply:
what do you think now? I also did some changes with Lawrence to the positioning of the link.